disperser/apiserver/server.go

```diff
@@ -839,7 +839,7 @@ func (s *DispersalServer) updateQuorumConfig(ctx context.Context) (QuorumConfig,
 
 func getResponseStatus(status disperser.BlobStatus) pb.BlobStatus {
 	switch status {
-	case disperser.Processing:
+	case disperser.Confirming, disperser.Processing:
```

Comment:
Why not return the CONFIRMING status?

Reply:
That would be a breaking change for the existing rollup integrations